forem/forem
# Better Error Msg when Listing URL embedded and Listings are disabled

## What type of PR is this? (check all applicable)

- [X] Improved User Experience

## Description
Now that Listings can be disabled on Forems, if a post with an embedded listing is re-saved, the resulting error message does not make intuitive sense. This PR presents a more explanatory error message.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/17111


## QA Instructions, Screenshots, Recordings
- Enable the `listing_feature_enabled` flag in `/admin/feature_flags/features` as an `admin`
- Create a post with an embedded listing URL 
- Disable the `listing_feature_enabled` flag in `/admin/feature_flags/features` as an `admin`
- Edit said post above and re-save. An error message should appear: `Listings are disabled on this Forem; cannot embed a listing URL`

### UI accessibility concerns?
None; all backend changes

## Added/updated tests?



## [Forem core team only] How will this change be communicated?
